public/index.html

```diff
@@ -37,15 +37,16 @@
 
                 <div class="content-box">
                     <p class="lead">
-                        Oleme Jaa !mproteater, spontaanne ning lõbus grupp noori amatöörnäitlejaid. Pakume publikule
-                        improviseeritud teatrietendustega rõõmu ja draamat ning õpetame huvilistele improviseerimist.
+                        Jaa !mproteater on Eesti teatrimaastikul tegutsenud aastast 2009 (enne 2016 nimega Improgrupp Jaa!). 
+                        Olles üks esimesi ja vanimaid improteatreid Eestis, on Jaa!l olnud määrav osa kohaliku improkultuuri arendamisel ja suunamisel.  
+                        Jaa! on andnud etendusi nii Eestis kui välismaal, toonud kohaliku publiku ette nii koolitajaid kui teatritruppe teistest riikidest ning korraldanud rahvusvahelist improteatrite festivali Tilt. 
+                        Etenduste formaatides pakub Jaa! publikule võimalust saada osa nii lühivormi sketšidest kui pikavormi etendustest. 
+                        Jaa !mproteater on alati valmis väljakutseteks ning otsib huviga uusi vorme ja võimalusi spontaansete teatrihetkede jagamiseks!
                     </p>
 
                     <p class="lead text-center links-group">
                         <a href="http://improv.ee/trupid/improgrupp-jaa/">
                             <strong>Mängukava</strong> </a> |
-                        <a href="http://kultuur.ee/index.php?option=com_content&amp;view=article&amp;id=569&amp;Itemid=620">
-                            <strong>Impro trennid</strong>
                         </a> |
```

Comment:
There's one extra "pipe" symbol between "Mängukava" and "Facebook" - perhaps that should be deleted as well?
![selection_002](https://cloud.githubusercontent.com/assets/469120/13390459/1fa3e51e-ded7-11e5-91e8-07a51f8b8702.png)